tests/test_unit_pto_detector_write_message.py

```diff
@@ -0,0 +1,32 @@
+import os, sys
+import boto3
+import conf.channel_conf as conf
+import warnings
+from unittest import mock
+from unittest.mock import patch
+from moto import mock_sqs
+from pto_detector import pto_detector
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Added following code to fix deprecation warning
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore",category=DeprecationWarning)
+    import imp
+
+@mock_sqs
+@patch('pto_detector.pto_detector.write_message')
```

Comment:
Same misconception with unit tests here too Rahul. You should not mock your method under test. You should instead mock all the methods that your method under test calls.

Reply:
@qxf2 , I have resolved your comments also, removed unittest dependency from the tests as discussed in sprint meeting. Can I share this for PR for review now?